Sources/Paywalls/PaywallData.swift

```diff
@@ -140,6 +148,32 @@ extension PaywallData {
 
 }
 
+extension PaywallData.LocalizedConfiguration {
+
+    /// An item to be showcased in a paywall.
+    public struct Feature {
```

Comment:
gotta love the ability to namespace things like this

Reply:
👍🏻  to avoid with a nested mess hehe